mod/messageboard/views/default/forms/messageboard/add.php

```diff
@@ -5,19 +5,25 @@
  * @package ElggMessageBoard
  */
 
-echo elgg_view('input/plaintext', array(
+
+echo elgg_view_field([
+	'#type' => 'plaintext',
```

Comment:
leaving this as is for now... don't want to surprise people with all kinds of longtext embed features in the messageboard